node/tessel-export.js

```diff
@@ -600,6 +600,15 @@ Tessel.Port.prototype._txrx = function(buf, cb) {
   this.uncork();
 };
 
+Tessel.Port.prototype.delay = function(delayUs) {
+  // Create a four byte buffer for the delay time
+  var microseconds = new Buffer(4);
+  // Pack the buffer with delay bytes
+  microseconds.writeUInt32BE(delayUs);
+  // Send off the delay command
+  this.sock.write(Buffer.concat([new Buffer([CMD.WAIT]), microseconds]));
```

Comment:
Instead of making two buffers, create a single 5 byte buffer and write the bytes into it: 

```js
var frame = new Buffer(5);
frame.writeUInt8(0xFF);
frame.writeUInt32BE(delayUs, 1);
 this.sock.write(frame);
```


Here's the difference when running on a Tessel 2 (1 execution each time): 

```
...
INFO Running measure-buffer-allocation.js...
title                              result
---------------------------------  ---------
Two Buffers, one write and concat  0s 20.7ms
One Buffer, two writes             0s 2.86ms
```

```
...
INFO Running measure-buffer-allocation.js...
title                              result
---------------------------------  ----------
Two Buffers, one write and concat  0s 20.66ms
One Buffer, two writes             0s 2.8ms
```

```
...
INFO Running measure-buffer-allocation.js...
title                              result
---------------------------------  ----------
Two Buffers, one write and concat  0s 20.82ms
One Buffer, two writes             0s 3.04ms
```




More importantly, this code must not be reachable if the delay is zero.